client/packages/requisitions/src/RequestRequisition/DetailView/Toolbar/Toolbar.tsx

```diff
@@ -101,6 +104,11 @@ export const Toolbar: FC = () => {
             />
           )}
         </Grid>
+        {programName && (
+          <Box padding={2}>
```

Comment:
A minor concern - the introduction of the info panel results in the toolbar taking up half the screen if the screen resolution is small / medium ( my laptop screen, for example ): 

<img width="1261" alt="image" src="https://user-images.githubusercontent.com/9192912/235382957-a482fb88-a7e6-4ec4-826f-755dfb382851.png">

```suggestion
          <Box padding={2} style={{ maxWidth: 500 }}>
```

and I've also added 

```
style={{ maxWidth: 375 }}
```

to the first column as well, so that there isn't a big blank space to the right of that column.

You could go totally nuts and change the margin on the `Grid` which contains the `SearchBar` to this:

```
        sx={{ marginTop: -4, marginBottom: -2 }}
```

which results in

<img width="1255" alt="image" src="https://user-images.githubusercontent.com/9192912/235383216-7b04a0f2-34fe-4401-bbdd-c457069abf61.png">